sfdo_template_helpers/admin/middleware.py

```diff
@@ -16,8 +16,15 @@ def __init__(self, get_response):
         self.ip_ranges = settings.ADMIN_API_ALLOWED_SUBNETS
 
     def __call__(self, request):
-        if request.path.startswith(f"/{settings.ADMIN_AREA_PREFIX}"):
-            self._validate_ip(request)
+        restricted_areas = tuple(getattr(settings, "RESTRICTED_PREFIXES", ()))
+        admin_area = getattr(settings, "ADMIN_AREA_PREFIX", None)
+        if admin_area is not None:
+            restricted_areas += (admin_area,)
```

Comment:
You can get rid of this conditional if you return an empty tuple instead of `None` as the default for the `getattr` call above.

Reply:
> You can get rid of this conditional if you return an empty tuple instead of `None` as the default for the `getattr` call above.

Actually, I thought that too, but it didn't work when I tried it. 

https://trinket.io/python/8b0fda5647

I mean it did "work" in the sense that the extra tuple didn't cause any problems at the end of the list, but it wasn't supposed to be there so some future refactor or type-strict version would run into problems.